ssl/record/ssl3_record.c

```diff
@@ -207,8 +210,32 @@ int ssl3_get_record(SSL *s)
             rret = ssl3_read_n(s, SSL3_RT_HEADER_LENGTH,
                                SSL3_BUFFER_get_len(rbuf), 0,
                                num_recs == 0 ? 1 : 0, &n);
-            if (rret <= 0)
-                return rret;     /* error or non-blocking */
+            if (rret <= 0) {
+                if (!BIO_get_ktls_recv(s->rbio))
+                    return rret;     /* error or non-blocking */
+#ifndef OPENSSL_NO_KTLS
+                switch (errno) {
+                case EBADMSG:
+                    SSLfatal(s, SSL_AD_BAD_RECORD_MAC,
+                            SSL_F_SSL3_GET_RECORD,
```

Comment:
please indent these these lines by one space more.

Reply:
Are you sure this is needed? in other places there is no indent of the "case" statements.

Reply:
i think @bernd-edlinger was referring to the line containing `SSL_F_SSL3_GET_RECORD`. Indentation is off-by-one.

Reply:
Oh I see. Fixed now.